### ISSUE TITLE
Missing import from IntervalArithmetic?

### ISSUE THREAD

**Issue by dpsanders** [JuliaIntervals/IntervalRootFinding.jl#200]
```
julia> using IntervalRootFinding
[ Info: Precompiling IntervalRootFinding [d2bf35a9-74e0-55ec-b149-d360ff49b807]
WARNING: could not import IntervalArithmetic.wideinterval into IntervalRootFinding
ERROR: LoadError: UndefVarError: `where_bisect` not defined
```

**Comment by XuJuan0505** [comment]
I met the same issue. Is it solved?

**Comment by Kolaru** [comment]
Not yet. The current progress in ensuring compatibility with the latest IA releases are in #203 (which got delayed)

**Comment by OlivierHnt** [comment]
Closed by #203.
